featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -120,7 +121,10 @@ def run(self, instance_ids):
         column_list = []
 
         # Order by instance_ids
-        df = df.reindex(list(pd.unique(instance_ids)))
+        unique_instance_ids = pd.unique(instance_ids)
```

Comment:
looks like before we call `FeatureSet.run()` some sorting of instance ids occurs. does it make sense to just remove that? 

https://github.com/Featuretools/featuretools/blob/5eef35b63823a257f04253130162f1338df5edee/featuretools/computational_backends/calculate_feature_matrix.py#L333

the other things, is we could do the conversion in the call there, as well. i'm don't have strong feelings either way though

Reply:
Some tests depend on that sorting being done. Should we guarantee the order?

Why would we need another type conversion?

Reply:
let's leave that sorting for now.

i should have said "type conversion in the call there, instead"